docs/pages/guides/web-performance.md

```diff
@@ -28,6 +28,17 @@ npx expo-optimize
 
 To inspect bundle sizes, you can use a Webpack plugin called [_Webpack Bundle Analyzer_](https://github.com/webpack-contrib/webpack-bundle-analyzer). This plugin will help you visualize the size of your static bundles. You can use this to identify unwanted large packages that you may not have bundled intentionally.
 
+
+### Using Bundle Analyzer for app bundle
+
+1. Install the source map explorer: `yarn add -D source-map-explorer`
+2. Create app export: `npx expo export --output-dir exported -p https://www.example.com/ --dump-sourcemap`.
```

Comment:
A more streamlined way to do this would be to use https://github.com/IjzerenHein/react-native-bundle-visualizer by @IjzerenHein

Reply:
I tried that package and it failed when running with `--expo managed` for my project. I just ran it again without `--expo` and it worked. So I can change the description, but maybe we should include these two steps as fallback.